azkaban_cli/tests/test_azkaban/test_resume_flow.py

```diff
@@ -0,0 +1,82 @@
+from unittest import TestCase
+from unittest.mock import patch, ANY
+
+import responses
+
+import azkaban_cli.azkaban
+from azkaban_cli.exceptions import ResumeFlowExecutionError, SessionError
+
+
+class AzkabanResumeFlowsTest(TestCase):
+    def setUp(self):
+        """
+        Creates an Azkaban instance and set a logged session for all upload tests
+        """
+
+        self.azk = azkaban_cli.azkaban.Azkaban()
+
+        self.host = 'http://azkaban-mock.com'
+        self.user = 'username'
+        self.session_id = 'aebe406b-d5e6-4056-add6-bf41091e42c6'
+
+        self.azk.set_logged_session(self.host, self.user, self.session_id)
+
+        self.exec_id = '1234'
+
+    @responses.activate
+    def test_resume_flows(self):
+        """
+        Test resume flows method from Azkaban class
+        """
+
+        responses.add(
+            responses.GET,
+            self.host + "/executor",
+            json={},
+            status=200
+        )
+
+        self.azk.resume_flow_execution(self.exec_id)
```

Comment:
This is failing the test in CI. It's raising an `Empty response` error. If an empty response is expected when things go right (the flow is resumed), then you shouldn't raise an error when this happens. I put a comment at the function to show you how to do it.

Reply:
Thanks for pointing it out.